src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -244,6 +270,29 @@ components:
         - status
         - sequenceNumber
 
+    CreateActionPlanRequest:
+      title: CreateActionPlanRequest
+      type: object
+      description: A request to create an  Action Plan with at least one or more Goals that a Prisoner aims to complete.
+      properties:
+        prisonNumber:
+          type: string
+          description: The ID of the prisoner
+          example: 'A1234BC'
+        reviewDate:
```

Comment:
I think we need to think this one through and understand it a little better ....

The proposed screens allow for entering a time range (0-3, 3,-6 months etc), or None, or a specific date.
Presumably we are thinking the UI typescript code (UI server side) will take the value from the user form object - eg: `3-6_MONTHS`, and work out an actual date, which will be used in this request object .

That works .... but .... will the CIAG ever need to see the original value again? Does it appear on a screen anywhere? Might it appear on an edit type screen at some point, where they CIAG can change the plan review date?
In this case we would need to know the original value (presumably?) so we can present a web screen that says "0-6 months".
I dont think we'll be able to reverse engineer / derive the original time frame, just from a date .... 🤔

Reply:
As discussed, I'm making a separate change to add an enum with 5 different values - e.g. `THREE_MONTHS`, `NO_REVIEW_DATE`, `SPECIFIC_DATE` - to cover the various options.

Reply:
I ended up getting rid of that enum! See the description of this PR: https://github.com/ministryofjustice/hmpps-education-and-work-plan-api/pull/68